ynh_add_swap/ynh_add_swap

```diff
@@ -63,3 +71,17 @@ ynh_del_swap () {
 		rm /swap_$app
 	fi
 }
+
+# Check if the device of the main mountpoint "/" is an SD card
+#
+# return 1 if it's an SD card, else 0
+_ynh_is_main_device_is_sd_card () {
+	local main_device=$(lsblk --output PKNAME --noheadings $(findmnt / --nofsroot --uniq --output source --noheadings --first-only))
+
+	if echo $main_device | grep --quiet "mmc" && [ $(cat /sys/block/$main_device/queue/rotational) -eq 0 ]
```

Comment:
Would be better to tail -n1 instead of cat, to be sure it wouldn't get a blank line at the end.
And to compare with ==, in case it's not just 1 or 0.